qcodes/instrument_drivers/Keysight/Keysight_N5173B.py

```diff
@@ -0,0 +1,4 @@
+from qcodes.instrument_drivers.Keysight.Keysight_N5183B import N5183B as N5173B
```

Comment:
this is good, but because it's really a different instrument i'd still suggest to implement it as a class:
```python
class N5173B(N5183B):
    pass
```
otherwise the 173 instances would actually be of class N5183B which i don't think is correct, even though the intention here is that the classes are the same

Reply:
That is a very good point, thank you!